package.js

```diff
@@ -1,18 +1,18 @@
 /* global Package */
 Package.describe({
-    name: 'socialize:likeable',
-    summary: 'A package implementing social "liking" or "starring"',
-    version: '1.0.0',
-    git: 'https://github.com/copleykj/socialize-likeable.git',
+  name: 'socialize:likeable',
+  summary: 'A package implementing social "liking" or "starring"',
+  version: '1.0.0',
+  git: 'https://github.com/copleykj/socialize-likeable.git'
 });
 
 Package.onUse(function _(api) {
-    api.versionsFrom('1.3');
+  api.versionsFrom('1.3');
 
-    api.use('socialize:linkable-model@1.0.0');
+  api.use(['socialize:linkable-model@1.0.0', 'socialize:user-model@1.0.0', 'reywood:publish-composite@1.5.2']);
```

Comment:
Good catch! We can also remove `linkable-model` here since `user-model` implies it.

Reply:
Oh, and user-model should be implied

Reply:
Done!